docs/tasks/upgrading/upgrading-0.10-0.11.rst

```diff
@@ -62,5 +74,23 @@ still contain an old annotation:
    Ingress resource "demo/testcrt contains old annotations: (certmanager.k8s.io/cluster-issuer)"
    Ingress resource "example/ingress-resource contains old annotations: (certmanager.k8s.io/cluster-issuer)"
 
+In order to help with this migration, the following CLI tool will automatically
+migrate these annotations for you. Note that it will not make any changes to
+your cluster for you.
+
+.. code-block:: shell
+
+   # Firstly, download the binary for your given platform
+   $ TODO: Add link
+
+   # Mark the binary as executable and run the binary against your cluster
+   $ chmod +x api-migration && ./api-migration --kubeconfig /path/to/my/kubeconfig
+
+   # Follow the CLI ouput and check for the difference that has been made in files
+   $ vim -d ingress.yaml ingress-migrated.yaml
```

Comment:
It'd probably be easier to refer people to `diff` instead of vim, as vim is about as popular as emacs 😉

Reply:
What's "emacs"?? :thinking: